app/Blockers.cs

```diff
@@ -0,0 +1,168 @@
+using Lms;
+using Lms.Models;
+using ConsoleTables;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Options;
+using System.Collections;
+using System.Windows.Markup;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+
+class Blockers : ICommand {
+    private LmsDbContext? db;
+
+    public Blockers() {
+        this.db =  null;
+    }
+
+    public Blockers(LmsDbContext db) {
+        this.db = db;
+    }
+
+    public string GetHelp() {
+        return $"""
+        Blockers
+
+        Description:
+        The blockers for assignments.
+        Verbs:
+        - list: {GetHelp(Verb.List)}
+        - create: {GetHelp(Verb.Create)}
+        - edit: {GetHelp(Verb.Edit)}
+        - delete: {GetHelp(Verb.Delete)}
+        """;
+    }
+
+    public string GetHelp(Verb verb) {
+        switch (verb) {
+            case Verb.List:
+                return "Lists the blockers for assignments.";
+            case Verb.Create:
+                return "Create a new blocker for a assignment.";
+            case Verb.Edit:
+                return "Edit the existing blocker for the assignment.";
+            case Verb.Delete:
+                return "Delete the existing blocker for the assignment.";
+            default:
+                throw new ArgumentException("Invalid verb.");
+        }
+    }
+
+    public void Execute(Verb verb, string[] args) {
+        switch (verb) {
+            // listing all blockers using ConsoleTable
+            case Verb.List:
+                var table = new ConsoleTable("Id", "Description", "WorkItems", "CreatedAt");
+
+                List<Block> blockers = db.Blockers.Include(b => b.WorkItems).ToList();
+                foreach (var block in blockers)
+                {
+                    string workItems = string.Join(", ", block.WorkItems.Select(w => w.Title));
+                    table.AddRow(block.Id, block.Description, workItems, block.CreatedAt);
+                }
+                table.Configure(o => o.NumberAlignment = Alignment.Right).Write(Format.Alternative);
```

Comment:
As you create unit tests, you'll want to move this code to a dedicate function to help with testing.

Reply:
Resolved :) good job!